tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -549,7 +558,7 @@ public function testCreateContributionChainedLineItems() {
       'trxn_id' => 12345,
       'invoice_id' => 67890,
       'source' => 'SSF',
-      'contribution_status_id' => 1,
+      'contribution_status_id' => 'Pending',
```

Comment:
Switch to pending to create a valid payment flow - does not affect what is being tested (which should never be done in code - use the order api)